requirements.txt

```diff
@@ -1,3 +1,3 @@
-pennylane>=0.7
+pennylane>=0.8
```

Comment:
This was extending the functionality of #21 with `QubitDevice` being used in this plugin.